### PR TITLE
Update component library version and add meeting events

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:**

**Description of changes:**
- Add meeting events component released in component library to console log meeting events through meeting lifecycle when received.
- Bump the component library version.

**Testing**

1. How did you test these changes?
- Run the react meeting demo.
- Join the meeting, you should see meeting events logged in the console.

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
The react meeting demo app.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/meeting/src/containers/MeetingEventObserver/index.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import { useMeetingEvent } from 'amazon-chime-sdk-component-library-react';
+
+const MeetingEventObserver = () => {
+  const meetingEvent = useMeetingEvent();
+  console.log('Received meeting event in MeetingEventObserver', meetingEvent);
+  return null;
+}
+
+export default MeetingEventObserver;
```
Comment (jing-chen1) [inline apps/meeting/src/containers/MeetingEventObserver/index.tsx:12]: [nit] new line at the end of file

File: apps/meeting/src/containers/MeetingEventObserver/index.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (xuesichao) [inline apps/meeting/src/containers/MeetingEventObserver/index.tsx:1]: Do we need to remove the year in this copyright? It's a new created file.

File: apps/meeting/src/containers/MeetingEventObserver/index.tsx (hunk 1 above)
Comment (xuesichao) [inline apps/meeting/src/containers/MeetingEventObserver/index.tsx:12]: nit: new line end of file.

File: apps/meeting/src/containers/MeetingEventObserver/index.tsx (hunk 2 above)
Comment (devalevenkatesh) [reply apps/meeting/src/containers/MeetingEventObserver/index.tsx]: Fixed it thanks for the catch!

File: apps/meeting/src/containers/MeetingEventObserver/index.tsx (hunk 1 above)
Comment (devalevenkatesh) [reply apps/meeting/src/containers/MeetingEventObserver/index.tsx]: Yup updated, I have to get some linter installed locally I think.

Comment (devalevenkatesh) [reply apps/meeting/src/containers/MeetingEventObserver/index.tsx]: Updated, Jing had the same comment.

**Review by xuesichao — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**